shared/common-adapters/web-view.js.flow

```diff
@@ -0,0 +1,15 @@
+// @flow
+import * as React from 'react'
+
+export type WebViewInjections = {
+  javaScript?: string,
+  css?: string,
+}
+
+export type WebViewProps = {
+  url: string,
+  injections?: WebViewInjections,
+  style?: Object,
+}
+
+declare export default React.ComponentType<WebViewProps>
```

Comment:
This component makes me a bit nervous...can you explain a bit if there are any risks with this?

Reply:
We are filtering returned mimeType in the KBFS http server to never return text/html or text/javascript, so there's that. We've also disabled download in electron. I think if we limit the use of this to internal URLs only it's probably fine. We also have the option to use segregated session for these webviews, which I haven't done but think is possible.

Are there specific risks you are thinking of?

Reply:
Ah, I made separate comments re. the injection of CSS. I took a closer look at how we interact with the http server and it seems ok, although I wonder if it would be clearer to pass keybase paths to `WebView` and have it do the path-to-URL conversion (but that can be in a separate PR)

Reply:
Good idea. Will do!

Reply:
@akalin-keybase I tried to have WebView do the URL converting, but it seems it's gonna be pretty complicated for some cases. For example the AVView uses a `about:blank` src, and use JavaScript to load the content dynamically. So either WebView should understand the video content, or it can't really control the URL being loaded. I guess having the WebView understand different types of content loaded is the right direction. What do you think?

(Either way it seems this is gonna have be a separate PR)